front/src/modules/settings/developers/types/ApisFieldItem.ts

```diff
@@ -0,0 +1,5 @@
+export type ApisFiedlItem = {
+  name: string;
+  type: 'internal' | 'published';
+  expiration: string;
```

Comment:
add an `id` to get a unique value to provide to `<SettingsApisFieldItemTableRow`